Slash/DB/MySQL/MySQL.pm

```diff
@@ -13652,22 +13652,16 @@ PRIMARY KEY (discussion_id, uid)
 		if (!$self->sqlDo("ALTER TABLE users_comments DROP mode")) {
                         return 0;
                 }
-		if (!$self->sqlDo("ALTER TABLE users_comments CHANGE mode_new mode ENUM('flat', 'nocomment', 'thread', 'threadtos', 'threadtng') NOT NULL DEFAULT 'thread'")) {
-                        return 0;
-                }
 		if (!$self->sqlDo("update users_comments set mode = 'threadtos' where mode = 'thread'")) {
```

Comment:
Good catch but what say we actually do it efficiently if we're going for efficiency. Check the new version.